src/main/resources/services/graphQL/generateGetLocalesField.es

```diff
@@ -0,0 +1,44 @@
+//import {toStr} from '@enonic/js-utils';
+
+import {
+	GraphQLString,
+	list
+} from '/lib/graphql';
+//import {getLocales as importedGetLocales} from '/lib/locales';
+import {getLocales as importedGetLocales} from '../../lib/locales';
+
+
+export function generateGetLocalesField({
+	glue
+}) {
+	return {
+		args: {
+			locale: GraphQLString,
+			query: GraphQLString
+		},
+		resolve: (env) => {
+			//log.info(`env:${toStr(env)}`);
+			const {
+				locale = undefined,
```

Comment:
Yeah, I don't remember why I did that. Looking at where the constant goes it looks like I could have used null instead. But since it's working, I'm not touching that now.